config.yaml

```diff
@@ -36,11 +36,6 @@ demo: &demo
           - # Test key #2, secret: fOhZOFr6SSnYv1/xfKPPMHdWLdQDboysJ4Q794Gezmw=
             - 3BAyX5pSiFerJgLsJZ9XyoBDMHPtq3NLqm5AGPcBpba6W644zSiSWy3vVr
             - 1000
-  witness:
-    db:
-      path: "/var/db/dscp-witness"
-      clean: false
-
     logging:
```

Comment:
I don't understand this change. This `logging` here should be a child of `witness` (see the educator component below), but with this it becomes a child of `core`.